src/geometry.rs

```diff
@@ -1722,4 +1772,109 @@ mod tests {
 
         assert!(geometry.has_no_orphan_vertices());
     }
+
+    #[test]
+    fn test_triangle_face_to_reverted_should_pass_because_compared_to_reverted() {
+        let face = TriangleFace::new_separate(1, 2, 3, 4, 5, 6);
+        let face_reverted_correct = TriangleFace::new_separate(3, 2, 1, 6, 5, 4);
+
+        let face_reverted_calculated = face.to_reverted();
+        assert_eq!(face_reverted_correct, face_reverted_calculated);
+    }
+
+    #[test]
+    fn test_triangle_face_to_reverted_should_fail_because_compared_to_same() {
```

Comment:
I can skip this test because what you describe is already covered by other tests. This one was just to make sure that the original and the reverted face are not considered equal. In fact, this is a test of equality rather than of the `to_reverted` function.